### PR TITLE
i18n get_locale without User

### LINKED ISSUE #544: I18nMiddleware.get_user_locale() cant get user for some actions (Poll closing)
When user closing Poll we get update without information that can give us current User object, so we cant get user.locale

```python:
async def get_user_locale(self, action: str, args: Tuple[Any]) -> str:
    user: types.User = types.User.get_current()
    locale: Locale = user.locale <--- raised exception here "NoneType" object 
                                      has no attribute 'locale'
```

how it can be fixed:
* adding check for this `pre_process_poll` action to I18nMiddleware.trigger()
`and  'poll' not in action`

maybe need investigation for other actions

aiogram 2.12.1
python 3.9.2
Manjaro 21.0

### PR DESCRIPTION
**Pull request by Olegt0rr**
# Description
Fixes #544 

## Type of change
- [х] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Operating System: 
* Python version: 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: aiogram/contrib/middlewares/i18n.py [hunk 1]
```diff
@@ -128,8 +128,8 @@ async def get_user_locale(self, action: str, args: Tuple[Any]) -> str:
         :param args: event arguments
         :return: locale name
```
Comment (uwinx) [inline aiogram/contrib/middlewares/i18n.py:129]: return in this function becomes `Optional[str]`

**Review by uwinx — CHANGES_REQUESTED** [review]

File: aiogram/contrib/middlewares/i18n.py [hunk 2]
```diff
@@ -122,7 +122,8 @@ def lazy_gettext(self, singular, plural=None, n=1, locale=None, enable_cache=Fal
     async def get_user_locale(self, action: str, args: Tuple[Any]) -> str:
```
Comment (uwinx) [inline aiogram/contrib/middlewares/i18n.py]:
```suggestion
    async def get_user_locale(self, action: str, args: Tuple[Any]) -> Optional[str]:
```

**Review by uwinx — CHANGES_REQUESTED** [review]

File: aiogram/contrib/middlewares/i18n.py (hunk 2 above)
Comment (Olegt0rr) [reply aiogram/contrib/middlewares/i18n.py]: Outdated code, already fixed
https://github.com/aiogram/aiogram/pull/546/files#diff-3eb6fb402a3689572f1a23bdb28610ddb0e2dcc5bd1ac4d3f5e24913706a5c16R122

File: aiogram/contrib/middlewares/i18n.py [hunk 3]
```diff
@@ -119,22 +119,24 @@ def lazy_gettext(self, singular, plural=None, n=1, locale=None, enable_cache=Fal
...
-        user: types.User = types.User.get_current()
-        locale: Locale = user.locale
+        user: Optional[types.User] = types.User.get_current()
+        locale: Optional[Locale] = user.locale if user else None
 
         if locale:
             *_, data = args
             language = data['locale'] = locale.language
             return language
+        return None
```
Comment (Termonoid) [inline aiogram/contrib/middlewares/i18n.py:139]: When method returns None, await on line 152 fails.
`locale = await self.get_user_locale(action, args)` <- awaiting None occurs here

Comment (Termonoid) [reply aiogram/contrib/middlewares/i18n.py:139]: We should check for None in `trigger` before awaiting result

**Review by Termonoid — CHANGES_REQUESTED** [review]

File: aiogram/contrib/middlewares/i18n.py (hunk 3 above)
Comment (uwinx) [reply aiogram/contrib/middlewares/i18n.py:139]: Please elaborate. we `await` for a coroutine and not the None. the `get_user_locale(...)` may return a `Awaitable[None | str]`, not `None`.

**Review by JrooTJunior — APPROVED** [review]

**Pull request merged**